test/hotspot/jtreg/runtime/valhalla/inlinetypes/ClassInitializationFailuresTest.java

```diff
@@ -103,45 +116,49 @@ static void testArrayInitialization() {
         Throwable e = null;
         try {
-            BadTwo[] array = new BadTwo[10];
+            BadTwo[] array = (BadTwo[])VM.newNullRestrictedArray(BadTwo.class, 10);
         } catch(Throwable t) {
             e = t;
         }
         Asserts.assertNotNull(e, "Error should have been thrown");
         Asserts.assertTrue(e.getClass() == ExceptionInInitializerError.class, " Must be an ExceptionInInitializerError");
         // Second attempt because it doesn't fail the same way
         try {
-            BadTwo[] array = new BadTwo[10];
+            BadTwo[] array = (BadTwo[])VM.newNullRestrictedArray(BadTwo.class, 10);
         } catch(Throwable t) {
             e = t;
         }
         Asserts.assertNotNull(e, "Error should have been thrown");
         Asserts.assertTrue(e.getClass() == NoClassDefFoundError.class, "Must be a NoClassDefFoundError");
         Asserts.assertTrue(e.getCause().getClass() == ExceptionInInitializerError.class, "Must be an ExceptionInInitializerError");
+        // Transition model (annotations and array factory) doesn't permit multi-dimentional arrays tests
+        // Disabling those tests for now
         // Testing multianewarray when the primitive element class fails to initialize properly
-        try {
-            BadThree[][] array = new BadThree[10][20];
-        } catch(Throwable t) {
-            e = t;
-        }
-        Asserts.assertNotNull(e, "Error should have been thrown");
-        Asserts.assertTrue(e.getClass() == ExceptionInInitializerError.class, " Must be an ExceptionInInitializerError");
-        // Second attempt because it doesn't fail the same way
-        try {
-            BadThree[][][] array = new BadThree[10][30][10];
-        } catch(Throwable t) {
-            e = t;
-        }
-        Asserts.assertNotNull(e, "Error should have been thrown");
-        Asserts.assertTrue(e.getClass() == NoClassDefFoundError.class, "Must be a NoClassDefFoundError");
-        Asserts.assertTrue(e.getCause().getClass() == ExceptionInInitializerError.class, "Must be an ExceptionInInitializerError");
+        // try {
+        //     BadThree[][] array = new BadThree[10][20];
+        // } catch(Throwable t) {
+        //     e = t;
+        // }
+        // Asserts.assertNotNull(e, "Error should have been thrown");
+        // Asserts.assertTrue(e.getClass() == ExceptionInInitializerError.class, " Must be an ExceptionInInitializerError");
+        // // Second attempt because it doesn't fail the same way
+        // try {
+        //     BadThree[][][] array = new BadThree[10][30][10];
+        // } catch(Throwable t) {
+        //     e = t;
+        // }
+        // Asserts.assertNotNull(e, "Error should have been thrown");
+        // Asserts.assertTrue(e.getClass() == NoClassDefFoundError.class, "Must be a NoClassDefFoundError");
+        // Asserts.assertTrue(e.getCause().getClass() == ExceptionInInitializerError.class, "Must be an ExceptionInInitializerError");
```

Comment:
I would suggest to use `/*` `*/` for uncommenting because it does not affect git blame history of the other lines.